content/docs/hooks-overview.md

```diff
@@ -201,9 +201,9 @@ function useFriendStatus(friendID) {
 }
 ```
 
-It takes `friendID` as an argument, and returns whether our friend is online.
+यह एक तर्क के रूप में `friendID` लेता है, और लौटता है कि क्या हमारा दोस्त ऑनलाइन है। यह एक आर्गुमेंट के रूप में `friendID` लेता है, और लौटता है कि क्या हमारा दोस्त ऑनलाइन है।
 
-Now we can use it from both components:
+अब हम इसे दोनों कौम्पोनॅन्टस का उपयोग कर सकते हैं:
```

Comment:
`का` => `से`

Reply:
@arshadkazmi42 
I think `मैं` makes much more sense here, what do you think?

the sentence will be: `अब हम इसे दोनों कौम्पोनॅन्टस मैं उपयोग कर सकते हैं`

Reply:
If you  will read the english version, it referring those components, `से` seems the right translation here.
Let me know if I am missing anything here.